pkg/transform/oauth/basicauth.go

```diff
@@ -44,12 +45,17 @@ func buildBasicAuthIP(serializer *json.Serializer, p IdentityProvider) (Identity
 	idP.MappingMethod = p.MappingMethod
 	idP.BasicAuth.URL = basicAuth.URL
 
+	if basicAuth.CA != "" {
+		caConfigmap = configmaps.GenConfigMap("basicauth-configmap", "openshift-config", p.CAData)
+		idP.BasicAuth.CA.Name = caConfigmap.Metadata.Name
+	}
+
 	certSecretName := p.Name + "-client-cert-secret"
 	idP.BasicAuth.TLSClientCert.Name = certSecretName
 	encoded := base64.StdEncoding.EncodeToString(p.CrtData)
 	certSecret, err = secrets.GenSecret(certSecretName, encoded, "openshift-config", "basicauth")
```

Comment:
I'm noticing the `openshift-config` namespace is hardcoded all over the place. Is it possible this value may differ between clusters? I.E., some clusters may have these secrets and certs in a different location? Regardless, this should be a const instead of a magic string.

Reply:
We can file an issue for this if we don't already have one, since it wasn't this PR that introduced that.

Reply:
"openshift-config" namespace is required for all configmaps and secrets related to oAuth https://docs.openshift.com/container-platform/4.1/authentication/identity_providers/configuring-ldap-identity-provider.html

Reply:
ack. It's still good practice to define this string value with a const and use that value as opposed to raw strings.